releasenotes/notes/fix-staticpod-uid-0348d0703c0d6961.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix uid of static pods to be unique across nodes and cluster.
```

Comment:
not sure it's worth adding release notes for now, cc @hkaj

Reply:
yeah no need for now, we'll see once more information about the feature is available publicly :)